cypress/plugins/seed.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-env node */
+
+const { spawn } = require('child_process');
+const fetch = require('node-fetch');
+
+async function runBundleExec(command) {
+  return new Promise((resolve, reject) => {
```

Comment:
Pulled out common functionality here. This will be where any Cypress task needs to tap into Rails.